sci-libs/gdal/gdal1-1.11.5.recipe

```diff
@@ -116,26 +130,72 @@ BUILD_PREREQUIRES="
 	cmd:pkg_config$secondaryArchSuffix
 	"
 
-defineDebugInfoPackage gdal1$secondaryArchSuffix \
-	$libDir/libgdal.so.$libVersion
+debugList=("$libDir"/libgdal.so.$libVersion)
+if [ -z "$commandSuffix" ]; then
+debugList+=(
+	"$commandBinDir"/gdal_contour \
+	"$commandBinDir"/gdal_grid \
+	"$commandBinDir"/gdal_rasterize \
+	"$commandBinDir"/gdal_translate \
+	"$commandBinDir"/gdaladdo \
+	"$commandBinDir"/gdalbuildvrt \
+	"$commandBinDir"/gdaldem \
+	"$commandBinDir"/gdalenhance \
+	"$commandBinDir"/gdalinfo \
+	"$commandBinDir"/gdallocationinfo \
+	"$commandBinDir"/gdalmanage \
+	"$commandBinDir"/gdalserver \
+	"$commandBinDir"/gdalsrsinfo \
+	"$commandBinDir"/gdaltindex \
+	"$commandBinDir"/gdaltransform \
+	"$commandBinDir"/gdalwarp \
+	"$commandBinDir"/nearblack \
+	"$commandBinDir"/ogr2ogr \
+	"$commandBinDir"/ogrinfo \
+	"$commandBinDir"/ogrlineref \
+	"$commandBinDir"/ogrtindex \
+	"$commandBinDir"/testepsg
+	)
+fi
+defineDebugInfoPackage gdal1$secondaryArchSuffix "${debugList[@]}"
 
 BUILD()
 {
-	expatDir=$portPackageLinksDir/lib~libexpat
-	autoreconf -fi
-	runConfigure ./configure --with-expat=$expatDir
+	autoreconf -vfi
+	LIBS="-Wl,--as-needed -lexpat" \
```

Comment:
I just did a clean build. Now I get a slightly different error:
* on x86_gcc2h:
```
gdal_contour --help
runtime_loader: /boot/system/lib/x86/libgdal.so.1.18.5: Could not resolve symbol 'connect'
resolve symbol "connect" returned: -2147478780
runtime_loader: /boot/system/lib/x86/libgdal.so.1.18.5: Troubles relocating: Symbol not found
```
* on x86_64:
```
gdal_contour --help
runtime_loader: /boot/system/lib/libgdal.so.1.18.5: Could not resolve symbol 'connect'
resolve symbol "connect" returned: -2147478780
runtime_loader: /boot/system/lib/libgdal.so.1.18.5: Troubles relocating: Symbol not found
```
So I guess this might work:
```diff
-	LIBS="-Wl,--as-needed -lexpat" \
+	LIBS="-Wl,--as-needed -lexpat -lnetwork" \
```

Reply:
Oops, it does get rid of the previous errors, but now it crashes.

<Details>

```
Debug information for team /boot/system/bin/gdal_contour (40166):
CPU(s): 1x Intel Core™ i5-6260U
Memory: 3.78 GiB total, 1.15 GiB used
Haiku revision: hrev52369 Sep 26 2018 21:43:21 (BePC)

Active Threads:
	thread 40167: team 40166 debug task 
	thread 40166: gdal_contour (main)
		state: Call (getNumAvailable() < getNumBlocks())

		Frame		IP			Function Name
		-----------------------------------------------
		00000000	0x65034112	commpage_syscall + 0x2 
			Disassembly:
				commpage_syscall:
				0x65034110:             89e1  mov %esp, %ecx
				0x65034112:             0f34  sysenter  <--

		0x71bc0008	0x17012e7	debugger + 0x31 
		0x71bc0028	0x170f7de	__assert_fail + 0x40 
		0x71bc0070	0x1785dc8	BPrivate::hoardHeap::freeBlock(BPrivate::block*&, BPrivate::superblock*&, int, BPrivate::processHeap*) + 0x278 
		0x71bc00d0	0x1786b80	BPrivate::processHeap::free(void*) + 0x244 
		0x71bc0100	0x17886e7	free + 0x54 
		0x71bc0118	0x1012b5b	VSIFree + 0x1b 
		0x71bc0158	0xff6cee	_ZL17CPLCleanupTLSListPPv.part.0 + 0x4e 
		0x71bc0198	0xfc2a5b	GDALDriverManager::~GDALDriverManager() + 0x16b 
		0x71bc01b8	0xfc2b27	GDALDriverManager::~GDALDriverManager() + 0x17 
		0x71bc01d8	0xfc365d	GDALDestroyDriverManager + 0x1d 
		0x71bc01e8	0xd6ab16	GDALDestroy() + 0x56 
		0x71bc0228	0x1b0f6a3	terminate_program + 0x93 
		0x71bc0240	0x1773ede	exit + 0x31 
		0x71bc0258	0x14e9c2b	/boot/system/bin/gdal_contour + 0x1c2b 
		0x71bc2238	0x14e9896	main + 0x936 
		0x71bc2278	0x14e9ac6	_start + 0x4b 
		0x71bc22a8	0x1b16e70	runtime_loader + 0x130 
		00000000	0x65034250	commpage_thread_exit + 0 

		Registers:
			  eip:	0x65034114
			  esp:	0x71bbffec
			  ebp:	0x71bc0008
			  eax:	0x000000e1
			  ebx:	0x017bd86c
			  ecx:	0x71bbffec
			  edx:	0x65034114
			  esi:	0x017af74c
			  edi:	0x181945c0
			   cs:	0x001b
			   ds:	0x0023
			   es:	0x0023
			   fs:	0x0063
			   gs:	0x0023
			   ss:	0x0023
			  st0:	0
			  st1:	0
			  st2:	0
			  st3:	0
			  st4:	0
			  st5:	0
			  st6:	9.53e+04
			  st7:	1e+03
			  mm0:	{0, 0, 0, 0}
			  mm1:	{0, 0, 0, 0}
			  mm2:	{0, 0, 0, 0}
			  mm3:	{0, 0, 0, 0}
			  mm4:	{0, 0, 0, 0}
			  mm5:	{0, 0, 0, 0}
			  mm6:	{0, 0, 0, 0xba18}
			  mm7:	{0, 0, 0, 0xfa00}
			 xmm0:	{0, 0, 0, 0, 0, 0, 0, 0}
			 xmm1:	{0, 0, 0, 0, 0, 0, 0, 0}
			 xmm2:	{0, 0, 0, 0, 0, 0, 0, 0}
			 xmm3:	{0, 0, 0, 0, 0, 0, 0, 0}
			 xmm4:	{0, 0, 0, 0, 0, 0, 0, 0}
			 xmm5:	{0, 0, 0, 0, 0, 0, 0, 0}
			 xmm6:	{0, 0, 0, 0, 0, 0, 0, 0}
			 xmm7:	{0, 0, 0, 0, 0, 0, 0, 0}

Loaded Images:
	ID		Text Base	Text End	Data Base	Data End	Type	Name
	--------------------------------------------------------------------------------
	491639	0x00595000	0x005f1000	0x005f1000	0x005f3000	lib    	/boot/system/lib/x86/libproj.so.13.1.1
	491636	0x007aa000	0x008fd000	0x008fd000	0x0090b000	lib    	/boot/system/lib/x86/libstdc++.so.6.0.24
	491634	0x00923000	0x00959000	0x00959000	0x00961000	lib    	/boot/system/lib/x86/libnetwork.so
	491632	0x00a0b000	0x00a23000	0x00a23000	0x00a24000	lib    	/boot/system/lib/x86/libz.so.1.2.11
	491631	0x00a9f000	0x00aca000	0x00aca000	0x00acb000	lib    	/boot/system/lib/x86/libpng16.so.16.35.0
	491626	0x00beb000	0x013c8000	0x013c8000	0x0145b000	lib    	/boot/system/lib/x86/libgdal.so.1.18.5
	491630	0x01468000	0x014ce000	0x014ce000	0x014d2000	lib    	/boot/system/lib/x86/libtiff.so.5.3.0
	491625	0x014e8000	0x014eb000	0x014eb000	0x014ec000	app    	/boot/system/bin/gdal_contour
	491637	0x015a9000	0x015c1000	0x015c1000	0x015c2000	lib    	/boot/system/lib/x86/libgcc_s.so.1
	491635	0x015f1000	0x016cd000	0x016cd000	0x016cf000	lib    	/boot/system/lib/x86/libiconv.so.2.5.0
	491638	0x016d9000	0x017ba000	0x017ba000	0x017fb000	lib    	/boot/system/lib/x86/libroot.so
	491633	0x01814000	0x01840000	0x01840000	0x01842000	lib    	/boot/system/lib/x86/libexpat.so.1.6.8
	491640	0x01a7c000	0x01a80000	0x01a80000	0x01a81000	lib    	/boot/system/lib/x86/libbsd.so
	491629	0x01a9d000	0x01ac0000	0x01ac0000	0x01ac6000	lib    	/boot/system/lib/x86/libgeotiff.so.2.1.2
	491624	0x01b05000	0x01b24000	0x01b24000	0x01b27000	lib    	/boot/system/runtime_loader
	491628	0x022c3000	0x022fc000	0x022fc000	0x022fe000	lib    	/boot/system/lib/x86/libjpeg.so.9.3.0
	491627	0x02553000	0x0255a000	0x0255a000	0x0255b000	lib    	/boot/system/lib/x86/libgif.so.7.0.0
	491623	0x65034000	0x6503c000	0x00000000	0x00000000	system 	commpage

Areas:
	ID		Base		End			Size (KiB)	Protection	Locking	Name
	--------------------------------------------------------------------------------
	1723564	0x00106000	0x00116000	        64	rw-        	full          	rld heap
	1723563	0x00117000	0x0011f000	        32	rw-        	full          	rld heap
	1723565	0x00129000	0x00139000	        64	rw-        	full          	rld heap
	1723525	0x001da000	0x001db000	         4	rw-S       	full          	_rld_debug_
	1723524	0x0031e000	0x0032e000	        64	rw-        	full          	rld heap
	1723559	0x00595000	0x005f1000	       368	r-x        	full          	libproj.so.13.1.1_seg0ro
	1723560	0x005f1000	0x005f3000	         8	rw-        	full          	libproj.so.13.1.1_seg1rw
	1723551	0x007aa000	0x008fd000	      1356	r-x        	full          	libstdc++.so.6.0.24_seg0ro
	1723552	0x008fd000	0x00904000	        28	rw-        	full          	libstdc++.so.6.0.24_seg1rw
	1723553	0x00904000	0x0090b000	        28	rw-        	full          	libstdc++.so.6.0.24_seg2rw
	1723546	0x00923000	0x00959000	       216	r-x        	full          	libnetwork.so_seg0ro
	1723547	0x00959000	0x0095c000	        12	rw-        	full          	libnetwork.so_seg1rw
	1723548	0x0095c000	0x00961000	        20	rw-        	full          	libnetwork.so_seg2rw
	1723542	0x00a0b000	0x00a23000	        96	r-x        	full          	libz.so.1.2.11_seg0ro
	1723543	0x00a23000	0x00a24000	         4	rw-        	full          	libz.so.1.2.11_seg1rw
	1723540	0x00a9f000	0x00aca000	       172	r-x        	full          	libpng16.so.16.35.0_seg0ro
	1723541	0x00aca000	0x00acb000	         4	rw-        	full          	libpng16.so.16.35.0_seg1rw
	1723528	0x00beb000	0x013c8000	      8052	r-x        	full          	libgdal.so.1.18.5_seg0ro
	1723529	0x013c8000	0x01459000	       580	rw-        	full          	libgdal.so.1.18.5_seg1rw
	1723530	0x01459000	0x0145b000	         8	rw-        	full          	libgdal.so.1.18.5_seg2rw
	1723538	0x01468000	0x014ce000	       408	r-x        	full          	libtiff.so.5.3.0_seg0ro
	1723539	0x014ce000	0x014d2000	        16	rw-        	full          	libtiff.so.5.3.0_seg1rw
	1723526	0x014e8000	0x014eb000	        12	r-x        	full          	gdal_contour_seg0ro
	1723527	0x014eb000	0x014ec000	         4	rw-        	full          	gdal_contour_seg1rw
	1723554	0x015a9000	0x015c1000	        96	r-x        	full          	libgcc_s.so.1_seg0ro
	1723555	0x015c1000	0x015c2000	         4	rw-        	full          	libgcc_s.so.1_seg1rw
	1723549	0x015f1000	0x016cd000	       880	r-x        	full          	libiconv.so.2.5.0_seg0ro
	1723550	0x016cd000	0x016cf000	         8	rw-        	full          	libiconv.so.2.5.0_seg1rw
	1723556	0x016d9000	0x017ba000	       900	r-x        	full          	libroot.so_seg0ro
	1723557	0x017ba000	0x017c4000	        40	rw-        	full          	libroot.so_seg1rw
	1723558	0x017c4000	0x017fb000	       220	rw-        	full          	libroot.so_seg2rw
	1723544	0x01814000	0x01840000	       176	r-x        	full          	libexpat.so.1.6.8_seg0ro
	1723545	0x01840000	0x01842000	         8	rw-        	full          	libexpat.so.1.6.8_seg1rw
	1723561	0x01a7c000	0x01a80000	        16	r-x        	full          	libbsd.so_seg0ro
	1723562	0x01a80000	0x01a81000	         4	rw-        	full          	libbsd.so_seg1rw
	1723535	0x01a9d000	0x01ac0000	       140	r-x        	full          	libgeotiff.so.2.1.2_seg0ro
	1723536	0x01ac0000	0x01ac5000	        20	rw-        	full          	libgeotiff.so.2.1.2_seg1rw
	1723537	0x01ac5000	0x01ac6000	         4	rw-        	full          	libgeotiff.so.2.1.2_seg2rw
	1723521	0x01b05000	0x01b24000	       124	r-x        	full          	runtime_loader_seg0ro
	1723522	0x01b24000	0x01b26000	         8	rw-        	full          	runtime_loader_seg1rw
	1723523	0x01b26000	0x01b27000	         4	rw-        	full          	runtime_loader_bss1
	1723533	0x022c3000	0x022fc000	       228	r-x        	full          	libjpeg.so.9.3.0_seg0ro
	1723534	0x022fc000	0x022fe000	         8	rw-        	full          	libjpeg.so.9.3.0_seg1rw
	1723531	0x02553000	0x0255a000	        28	r-x        	full          	libgif.so.7.0.0_seg0ro
	1723532	0x0255a000	0x0255b000	         4	rw-        	full          	libgif.so.7.0.0_seg1rw
	1723566	0x18193000	0x181f3000	       384	rw-        	full          	heap
	1723518	0x60cc2000	0x60cc6000	        16	rw-        	full          	user area
	1723520	0x65034000	0x6503c000	        32	r-xSk      	full          	commpage
	1723519	0x70bbf000	0x71bc5000	     16408	rw-s       	full          	gdal_contour_40166_stack

Protection Flags: r - read, w - write, x - execute, s - stack, o - overcommit, c - cloneable, S - shared, k - kernel

Semaphores:
	ID		Count	Last Holder	Name
	------------------------------------------------------------
```
</Details>

(that's on x86_gcc2 hrev52369)

<Details>

```
Debug information for team /boot/system/bin/gdal_contour (39160):
CPU(s): 1x Intel Core™ i5-6260U
Memory: 3.78 GiB total, 1.60 GiB used
Haiku revision: hrev52369 Sep 26 2018 21:53:15 (x86_64)

Active Threads:
	thread 39161: team 39160 debug task 
	thread 39160: gdal_contour (main)
		state: Call (getNumAvailable() < getNumBlocks())

		Frame		IP			Function Name
		-----------------------------------------------
		00000000	0x15bee0f6aa7	_kern_debugger + 0x7 
			Disassembly:
				_kern_debugger:
				0x0000015bee0f6aa0:   48c7c0e1000000  mov $0xe1, %rax
				0x0000015bee0f6aa7:             0f05  syscall  <--

		0x7ffb76fdaf80	0x15bee0f01c5	debugger + 0x2f 
		0x7ffb76fdafb0	0x15bee0fd13f	__assert_fail + 0x3e 
		0x7ffb76fdb000	0x15bee19a956	BPrivate::hoardHeap::freeBlock(BPrivate::block*&, BPrivate::superblock*&, int, BPrivate::processHeap*) + 0x278 
		0x7ffb76fdb040	0x15bee19b64b	BPrivate::processHeap::free(void*) + 0x23d 
		0x7ffb76fdb060	0x15bee19cf55	free + 0x3d 
		0x7ffb76fdb0a0	0x1ec0bd2ca57	/boot/system/lib/libgdal.so.1.18.5 + 0x4e1a57 
		0x7ffb76fdb0c0	0x1ec0bd2cb01	/boot/system/lib/libgdal.so.1.18.5 + 0x4e1b01 
		0x7ffb76fdb0d0	0x1ec0baea709	/boot/system/lib/libgdal.so.1.18.5 + 0x29f709 
		0x7ffb76fdb0f0	0x1ec0baead07	/boot/system/lib/libgdal.so.1.18.5 + 0x29fd07 
		0x7ffb76fdb110	0x1ec0bfa46e5	/boot/system/lib/libgdal.so.1.18.5 + 0x7596e5 
		0x7ffb76fdb160	0x562a3ea4a1	terminate_program + 0x6d 
		0x7ffb76fdb180	0x15bee188ca6	exit + 0x29 
		0x7ffb76fdb190	0x25e950380a2	/boot/system/bin/gdal_contour + 0x20a2 
		0x7ffb76fdd1b0	0x25e95037d54	main + 0x884 
		0x7ffb76fdd1e0	0x25e95037f89	_start + 0x4f 
		0x7ffb76fdd220	0x562a3f189e	runtime_loader + 0x105 
		00000000	0x7fffffd24260	commpage_thread_exit + 0 

		Registers:
			  rip:	0x0000015bee0f6aa9
			  rsp:	0x00007ffb76fdaf58
			  rbp:	0x00007ffb76fdaf70
			  rax:	0x00000000000000e1
			  rbx:	0x0000015bee1fa020
			  rcx:	0x0000015bee0f6aa9
			  rdx:	0x00007ffb76fdaa94
			  rsi:	0x0000015bee1ab4a5
			  rdi:	0x0000015bee1fa020
			   r8:	0x0000000000000000
			   r9:	0x0000015bee1f91e0
			  r10:	0x00000000000000bf
			  r11:	0x0000000000003202
			  r12:	0x0000000000000013
			  r13:	0x000000c1cda09610
			  r14:	0x000000c1cda09610
			  r15:	0x00007ffb76fdb000
			   cs:	0x002b
			   ds:	0x0000
			   es:	0x0000
			   fs:	0x0000
			   gs:	0x0000
			   ss:	0x0023
			  st0:	0
			  st1:	0
			  st2:	0
			  st3:	0
			  st4:	0
			  st5:	0
			  st6:	0
			  st7:	0
			  mm0:	{0, 0, 0, 0}
			  mm1:	{0, 0, 0, 0}
			  mm2:	{0, 0, 0, 0}
			  mm3:	{0, 0, 0, 0}
			  mm4:	{0, 0, 0, 0}
			  mm5:	{0, 0, 0, 0}
			  mm6:	{0, 0, 0, 0}
			  mm7:	{0, 0, 0, 0}
			 xmm0:	{0, 0, 0, 0, 0, 0, 0, 0}
			 xmm1:	{0, 0, 0, 0, 0, 0, 0, 0}
			 xmm2:	{0, 0, 0, 0, 0, 0, 0, 0}
			 xmm3:	{0, 0, 0, 0, 0, 0, 0, 0}
			 xmm4:	{0, 0, 0, 0, 0, 0, 0, 0}
			 xmm5:	{0, 0, 0, 0, 0, 0, 0, 0}
			 xmm6:	{0, 0, 0, 0, 0, 0, 0, 0}
			 xmm7:	{0, 0, 0, 0, 0, 0, 0, 0}
			 xmm8:	{0, 0, 0, 0, 0, 0, 0, 0}
			 xmm9:	{0, 0, 0, 0, 0, 0, 0, 0}
			xmm10:	{0, 0, 0, 0, 0, 0, 0, 0}
			xmm11:	{0, 0, 0, 0, 0, 0, 0, 0}
			xmm12:	{0, 0, 0, 0, 0, 0, 0, 0}
			xmm13:	{0, 0, 0, 0, 0, 0, 0, 0}
			xmm14:	{0, 0, 0, 0, 0, 0, 0, 0}
			xmm15:	{0, 0, 0, 0, 0, 0, 0, 0}

Loaded Images:
	ID		Text Base	Text End	Data Base	Data End	Type	Name
	--------------------------------------------------------------------------------
	523390	0x562a3de000	0x562a3fe000	0x562a5fd000	0x562a600000	lib    	/boot/system/runtime_loader
	523402	0xd48cc31000	0xd48cd99000	0xd48cf99000	0xd48cfad000	lib    	/boot/system/lib/libstdc++.so.6.0.24
	523401	0xd52135e000	0xd521442000	0xd521641000	0xd521643000	lib    	/boot/system/lib/libiconv.so.2.6.0
	523399	0xe58fe9f000	0xe58fece000	0xe5900ce000	0xe5900d0000	lib    	/boot/system/lib/libexpat.so.1.6.8
	523398	0x1085e036000	0x1085e052000	0x1085e251000	0x1085e252000	lib    	/boot/system/lib/libz.so.1.2.11
	523406	0x122ce292000	0x122ce297000	0x122ce496000	0x122ce497000	lib    	/boot/system/lib/libbsd.so
	523405	0x14bf21cf000	0x14bf223f000	0x14bf243e000	0x14bf2442000	lib    	/boot/system/lib/libproj.so.13.1.1
	523404	0x15bee0b6000	0x15bee212000	0x15bee412000	0x15bee465000	lib    	/boot/system/lib/libroot.so
	523403	0x1a6c3f73000	0x1a6c3f88000	0x1a6c4187000	0x1a6c4188000	lib    	/boot/system/lib/libgcc_s.so.1
	523400	0x1b109779000	0x1b1097b9000	0x1b1099b8000	0x1b1099c1000	lib    	/boot/system/lib/libnetwork.so
	523396	0x1b1ee401000	0x1b1ee471000	0x1b1ee670000	0x1b1ee674000	lib    	/boot/system/lib/libtiff.so.5.3.0
	523397	0x1caa77a0000	0x1caa77d1000	0x1caa79d1000	0x1caa79d2000	lib    	/boot/system/lib/libpng16.so.16.35.0
	523395	0x1d0250b2000	0x1d0250df000	0x1d0252de000	0x1d0252e8000	lib    	/boot/system/lib/libgeotiff.so.2.1.2
	523394	0x1d0bea72000	0x1d0beaab000	0x1d0becab000	0x1d0becac000	lib    	/boot/system/lib/libjpeg.so.9.3.0
	523392	0x1ec0b84b000	0x1ec0c116000	0x1ec0c315000	0x1ec0c3f8000	lib    	/boot/system/lib/libgdal.so.1.18.5
	523393	0x255a4d15000	0x255a4d1d000	0x255a4f1c000	0x255a4f1d000	lib    	/boot/system/lib/libgif.so.7.0.0
	523391	0x25e95036000	0x25e95039000	0x25e95238000	0x25e95239000	app    	/boot/system/bin/gdal_contour
	523389	0x7fffffd24000	0x7fffffd2c000	0x00000000	0x00000000	system 	commpage

Areas:
	ID		Base		End			Size (KiB)	Protection	Locking	Name
	--------------------------------------------------------------------------------
	1680935	0x00454000	0x00474000	       128	rw-        	full          	rld heap
	1680934	0x00824000	0x00844000	       128	rw-        	full          	rld heap
	1680933	0x35222000	0x35232000	        64	rw-        	full          	rld heap
	1680932	0x1142e7000	0x1142f7000	        64	rw-        	full          	rld heap
	1680931	0x5105f2000	0x5105fa000	        32	rw-        	full          	rld heap
	1680893	0x76f67f000	0x76f680000	         4	rw-S       	full          	_rld_debug_
	1680892	0x2056e6b000	0x2056e7b000	        64	rw-        	full          	rld heap
	1680890	0x562a3de000	0x562a3fe000	       128	r-x        	full          	runtime_loader_seg0ro
	1680891	0x562a5fd000	0x562a600000	        12	rw-        	full          	runtime_loader_seg1rw
	1680936	0xc1cda07000	0xc1cda87000	       512	rw-        	full          	heap
	1680919	0xd48cc31000	0xd48cd99000	      1440	r-x        	full          	libstdc++.so.6.0.24_seg0ro
	1680920	0xd48cf99000	0xd48cfa5000	        48	rw-        	full          	libstdc++.so.6.0.24_seg1rw
	1680921	0xd48cfa5000	0xd48cfad000	        32	rw-        	full          	libstdc++.so.6.0.24_seg2rw
	1680917	0xd52135e000	0xd521442000	       912	r-x        	full          	libiconv.so.2.6.0_seg0ro
	1680918	0xd521641000	0xd521643000	         8	rw-        	full          	libiconv.so.2.6.0_seg1rw
	1680912	0xe58fe9f000	0xe58fece000	       188	r-x        	full          	libexpat.so.1.6.8_seg0ro
	1680913	0xe5900ce000	0xe5900d0000	         8	rw-        	full          	libexpat.so.1.6.8_seg1rw
	1680910	0x1085e036000	0x1085e052000	       112	r-x        	full          	libz.so.1.2.11_seg0ro
	1680911	0x1085e251000	0x1085e252000	         4	rw-        	full          	libz.so.1.2.11_seg1rw
	1680929	0x122ce292000	0x122ce297000	        20	r-x        	full          	libbsd.so_seg0ro
	1680930	0x122ce496000	0x122ce497000	         4	rw-        	full          	libbsd.so_seg1rw
	1680927	0x14bf21cf000	0x14bf223f000	       448	r-x        	full          	libproj.so.13.1.1_seg0ro
	1680928	0x14bf243e000	0x14bf2442000	        16	rw-        	full          	libproj.so.13.1.1_seg1rw
	1680924	0x15bee0b6000	0x15bee212000	      1392	r-x        	full          	libroot.so_seg0ro
	1680925	0x15bee412000	0x15bee420000	        56	rw-        	full          	libroot.so_seg1rw
	1680926	0x15bee420000	0x15bee465000	       276	rw-        	full          	libroot.so_seg2rw
	1680922	0x1a6c3f73000	0x1a6c3f88000	        84	r-x        	full          	libgcc_s.so.1_seg0ro
	1680923	0x1a6c4187000	0x1a6c4188000	         4	rw-        	full          	libgcc_s.so.1_seg1rw
	1680914	0x1b109779000	0x1b1097b9000	       256	r-x        	full          	libnetwork.so_seg0ro
	1680915	0x1b1099b8000	0x1b1099bb000	        12	rw-        	full          	libnetwork.so_seg1rw
	1680916	0x1b1099bb000	0x1b1099c1000	        24	rw-        	full          	libnetwork.so_seg2rw
	1680906	0x1b1ee401000	0x1b1ee471000	       448	r-x        	full          	libtiff.so.5.3.0_seg0ro
	1680907	0x1b1ee670000	0x1b1ee674000	        16	rw-        	full          	libtiff.so.5.3.0_seg1rw
	1680908	0x1caa77a0000	0x1caa77d1000	       196	r-x        	full          	libpng16.so.16.35.0_seg0ro
	1680909	0x1caa79d1000	0x1caa79d2000	         4	rw-        	full          	libpng16.so.16.35.0_seg1rw
	1680903	0x1d0250b2000	0x1d0250df000	       180	r-x        	full          	libgeotiff.so.2.1.2_seg0ro
	1680904	0x1d0252de000	0x1d0252e7000	        36	rw-        	full          	libgeotiff.so.2.1.2_seg1rw
	1680905	0x1d0252e7000	0x1d0252e8000	         4	rw-        	full          	libgeotiff.so.2.1.2_seg2rw
	1680901	0x1d0bea72000	0x1d0beaab000	       228	r-x        	full          	libjpeg.so.9.3.0_seg0ro
	1680902	0x1d0becab000	0x1d0becac000	         4	rw-        	full          	libjpeg.so.9.3.0_seg1rw
	1680896	0x1ec0b84b000	0x1ec0c116000	      9004	r-x        	full          	libgdal.so.1.18.5_seg0ro
	1680897	0x1ec0c315000	0x1ec0c3f6000	       900	rw-        	full          	libgdal.so.1.18.5_seg1rw
	1680898	0x1ec0c3f6000	0x1ec0c3f8000	         8	rw-        	full          	libgdal.so.1.18.5_seg2rw
	1680899	0x255a4d15000	0x255a4d1d000	        32	r-x        	full          	libgif.so.7.0.0_seg0ro
	1680900	0x255a4f1c000	0x255a4f1d000	         4	rw-        	full          	libgif.so.7.0.0_seg1rw
	1680894	0x25e95036000	0x25e95039000	        12	r-x        	full          	gdal_contour_seg0ro
	1680895	0x25e95238000	0x25e95239000	         4	rw-        	full          	gdal_contour_seg1rw
	1680887	0x7fd4dec09000	0x7fd4dec0d000	        16	rw-        	full          	user area
	1680888	0x7ffb75fda000	0x7ffb76fe0000	     16408	rw-s       	full          	gdal_contour_39160_stack
	1680889	0x7fffffd24000	0x7fffffd2c000	        32	r-xSk      	full          	commpage

Protection Flags: r - read, w - write, x - execute, s - stack, o - overcommit, c - cloneable, S - shared, k - kernel

Semaphores:
	ID		Count	Last Holder	Name
	------------------------------------------------------------
```
</Details>

(that's on x86_64 hrev52369)

Well, maybe I should retry with a clean build.

Reply:
Same as I experienced here, but for some reason after switching repos I can't build it anymore?
*** Failed to find a match for "haiku_x86 >= r1~beta1_hrev52358-1": Name not found

Reply:
That crash might be Haiku's problem, given that it happens in `libroot`... :/ @waddlesplash care to check?